src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/SimpleRecordsFetcherFactory.java

```diff
@@ -21,8 +21,8 @@
 @CommonsLog
 public class SimpleRecordsFetcherFactory implements RecordsFetcherFactory {
```

Comment:
Even though this is unchanged: 
Would it make more sense to have a records fetcher factory for each fetching strategy?  The Blocking strategy doesn't care about any of the configuration.